server/schedule/filter/metrics.go

```diff
@@ -23,7 +23,7 @@ var (
 			Subsystem: "schedule",
 			Name:      "filter",
 			Help:      "Counter of the filter",
-		}, []string{"action", "address", "store", "scope", "type", "source", "target"})
+		}, []string{"action", "scope", "type", "store"})
```

Comment:
The prom query also needs to be changed.

Reply:
the sub pr is: https://github.com/tikv/pd/pull/5577